docs/usage_guide/optimization_guide/index.rst

```diff
@@ -0,0 +1,58 @@
+Optimization Guide
+==================
+
+Here you can find information on Modin optimizations both for general query pipeline and for specific operations.
+
+Generic settings
+""""""""""""""""
+
+If you are working with a wide DataFrame and performing an operation that requires ``map`` Core Dataframe Algebra operator
+to be applied cell-wise, you may want to change the number of partitions DataFrame should partition into to avoid
+excessive amount of remote partitions, which is much greater than the number of CPUs.
```

Comment:
```suggestion
to be applied cell-wise, you may want to change the number of DataFrame partitions to avoid
an excessive amount of remote partitions, which is much greater than the number of CPUs.
```